Cargo.toml

```diff
@@ -200,6 +200,16 @@ description = "Renders a sprite"
 category = "2D Rendering"
 wasm = true
 
+[[example]]
+name = "sprite_slice"
+path = "examples/2d/sprite_slice.rs"
+
+[package.metadata.example.sprite_slice]
+name = "Sprite Slice"
+description = "Showcases sprite slicing"
```

Comment:
```suggestion
description = "Showcases slicing sprites into sections that can be scaled independently via the 9-patch technique"
```

Reply:
Try to avoid saying the same thing in the same words :) If you can explain it differently you may help explain the concept to different audience members.